devices/Ina219/README.md

```diff
@@ -53,5 +58,3 @@ at 5mV. Given this we can use a shunt voltage range of +/- 40mV
 * The maximum possible current would then be 40mV / 0.1 = 400mA
 * With a 400mA maximum current and a range of the ADC of 15bits then the LSB of the current would be 400mA/32767 = 12.2207 microamps. We will chose 12.2uA as a round number.
 * From the [INA219 Datasheet](http://www.ti.com/lit/ds/symlink/ina219.pdf) the calibration register should be set at 0.04096/(currentLSB * shunt resistance) = 33574 = 0x8326
-
-![circuit](Ina219.Sample_bb.png)
```

Comment:
I saw you added a fritzing diagram, I guess it's to place here, right?

Reply:
@rschiefer care to take care of this tiny nit so this can be merged? 😉

Reply:
Sorry, I missed this.  Yes, I will fix it